presentation/src/main/java/com/going/presentation/todo/ourtodo/checkfriends/CheckFriendsActivity.kt

```diff
@@ -203,17 +231,22 @@ class CheckFriendsActivity :
     override fun onDestroy() {
         super.onDestroy()
         _adapter = null
+        if (friendInviteDialog?.isAdded == true) friendInviteDialog?.dismiss()
```

Comment:
`?? == ture` 라면 뒷부분은 날려도 될듯~~

```suggestion
        if (friendInviteDialog?.isAdded) friendInviteDialog?.dismiss()
```

Reply:
현재 `FriendInviteEmptyDialog`가 nullable인데  ` == true`를 제거하면 if문은 Boolean 타입인데 Boolean? 타입이 들어가서 타입 불일치가 일어나는 것 같습니다,,!